docs/4.0/aws_oss_guide.md

```diff
@@ -0,0 +1,342 @@
+# Running Teleport on AWS
+
+We've created this guide to give customers a high level overview of how to use Teleport
+on Amazon Web Services (AWS). This guide starts with a high level introduction and 
+with a deeper dive into how to setup Teleport and run Teleport in production. 
+We have split this guide into. 
+
+- [Teleport on AWS FAQ](#teleport-on-aws-faq)
+- [Setting up Teleport OSS on AWS](#setting-up-teleport-oss-on-aws)
+- [Setting up Teleport Enterprise on AWS](#running-teleport-enterprise-on-aws)
+- [Teleport AWS Tips & Tricks](#teleport-aws-tips-tricks)
+
+### Teleport on AWS FAQ
+**Why would you want to use Teleport with AWS?**
+
+At some point you'll want to log into the system using SSH
+to help test, debug and troubleshoot a problem box. For EC2, AWS recommends creating 
+['Key Pairs'](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) 
+and has a range of [other tips for securing EC2 instances](https://aws.amazon.com/articles/tips-for-securing-your-ec2-instance/).
+
+There is a range of limits with this setup. 
+
+1. As your organization grows keeping track of end users public/private keys becomes
+   an administrative nightmare. 
+2. Using SSH public/private keys has a range of limits. Read why [SSH Certificates are better](https://gravitational.com/blog/ssh-key-management/). 
+3. Once a machine has been bootstrapped with SSH Keys, there isn't an easy way to 
+  add new keys and delegate access. 
+
+**Which Services can I use Teleport with?**
+
+You can use Teleport for all the services that you would SSH into. This guide is focused
+on EC2. We have a short blog post on using Teleport with [EKS](https://gravitational.com/blog/teleport-aws-eks/). We plan to expand the guide based on feedback but will plan to add instructions
+for the below. 
+
+- RDS
+- Detailed EKS
+- Lightsail
+- Fargate
+- AWS ECS 
+
+## Teleport Introduction
+
+This guide will cover how to setup, configure and run Teleport on [AWS](https://aws.amazon.com/). 
+ 
+#### AWS Services required to run Teleport in HA  
+
+- [EC2 / Autoscale](#ec2-autoscale)
+- [DynamoDB](#dynamodb)
+- [S3](#s3)
+- [Route53](#route53) 
+- [NLB](#nlb-network-load-balancer)
+- [IAM](#iam)
+- [ACM](#acm)
+- [SSM](#aws-systems-manager-parameter-store) 
+
+We recommend setting up Teleport in high availability mode (HA). In HA mode DynamoDB 
+stores the state of the system and S3 will store audit logs. 
+
+![AWS Intro Image](img/aws/aws-intro.png)
+
+### EC2 / Autoscale
+To run Teleport in a HA configuration we recommend using m4.large instances and it's best
+practice to separate the proxy and authentication server, creating auto-scale groups 
```

Comment:
I think AWS uses the term `autoscaling` in all their documentation - might be worth sticking to that for all mentions in the guide.